consul_exporter.go

```diff
@@ -86,6 +86,15 @@ func NewExporter(uri string) *Exporter {
 			[]string{"service", "node"},
 		),
 
+		nodeChecks: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Namespace: namespace,
+				Name:      "agent_check",
+				Help:      "Is this check passing on this node?",
```

Comment:
Oh. Just noticed you're using a label to indicate the status of the check. That's not so easy to work with in expressions in the end. The better way would be to omit the status label and set the value of the time series to 0 or 1 (like we do with the synthetic `up` metric).